### ISSUE TITLE
VAST on Android default broswer

### ISSUE THREAD

**Issue by Granze** [PetroFrolov/vast-video-js#14]
I'm trying to run the plugin in the default android browser but it doesn't work. I don't get any kind of error but the ad video doesn't start.

It works fine under chrome mobile.

**Comment by PetroFrolov** [comment]
Hi! What version of Android do you use?

I have 2.2 and 2.3.
On my devices video didn't open in browser, it playing by default video player, so ... the JS don't work.

**Comment by Granze** [comment]
Hey!
My main test device is a 4.2.2 but I did some test on others 4.x without success.

**Comment by PetroFrolov** [comment]
I've tested on Sony Xpreia with 4.0.4 and in Chroe mobile I'd seen only post-roll. Pre-roll wasn't show (
In default browser player isn't liook like video.js player (no skin), so I think that video.js don't work in it.
Why? I don't know. It's too difficult to understand behavior without debuging, but default browser doesn't have developers tools

**Comment by Granze** [comment]
yes, trying to figure out what happens in the default browser is a pain.

Thanks anyway!

**Comment by D3adKnight** [comment]
In our case, the problem with Android was due to "untimely" determine the duration of the video, and as a consequence of incorrect start time for putting "post-roll" Ad. Try to "catch" him when it start playing and the correct his timing.

**Comment by PetroFrolov** [comment]
Hi! Yes, post-roll show before end of video because its the simplest way. It will be fixed later.
Currently I am trying to rewrite plugin for the last version of VideoJS but there are some issues.

**Comment by D3adKnight** [comment]
No, in your algorithm all fine.
Troubles in Android. It's incorrect detects 'duraton' of video (usually I have 1 sec) and then 'post-roll' plays immidiatly after video start. And then current rolls index already has max value
